wagtailio/static/sass/components/_cta.scss

```diff
@@ -0,0 +1,46 @@
+.cta {
+    $root: &;
+    padding: 25px 30px;
```

Comment:
What do you think about dropping this to `15px 20px` on mobile?

![image](https://user-images.githubusercontent.com/31627284/181254462-7bfda351-b33f-4ec4-8ec9-a83a9514537a.png)

Reply:
Yeah, that looks cleaner.